.travis.yml

```diff
@@ -4,12 +4,12 @@ install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.t
 script: bash -ex .travis-opam.sh
 env:
   - OCAML_VERSION=latest TESTS=true PACKAGE=ocplib-endian
```

Comment:
This line should be removed since ocaml-ci-scripts removed the support for "latest" version.